src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java

```diff
@@ -159,7 +159,7 @@
                 .setParameter("id", id)
                 .setParameter("accountId", tenant)
                 .getSingleResultOrNull()
-                .onItem().ifNotNull().transformToUni(this::loadProperties);
+                .onItem().ifNotNull().transformToUni(endpoint -> loadProperties(endpoint));
```

Comment:
This may look unnecessary but the compiler wasn't happy without it:

```
reference to transformToUni is ambiguous
  both method <R>transformToUni(java.util.function.Function<? super T,io.smallrye.mutiny.Uni<? extends R>>) in io.smallrye.mutiny.groups.UniOnNotNull and method <R>transformToUni(java.util.function.BiConsumer<? super T,io.smallrye.mutiny.subscription.UniEmitter<? super R>>) in io.smallrye.mutiny.groups.UniOnNotNull match
```